addons/web_editor/static/lib/odoo-editor/src/powerbox/Powerbox.js

```diff
@@ -254,10 +260,12 @@ export class Powerbox {
 
             this.options.onStop && this.options.onStop();
         };
-        this._currentValidate = () => {
-            const command = this._currentFilteredCommands.find(
-                c => c === this._currentSelectedCommand,
-            );
+        this._currentValidate = (command) => {
+            if (!command) {
+                command = this._currentFilteredCommands.find(
+                    c => c === this._currentSelectedCommand,
+                );
+            }
```

Comment:
This part matches the commit message, the rest doesn't and should be with the previous commit.

Reply:
true, solved